components/Common/Search/index.module.css

```diff
@@ -23,3 +23,12 @@
     top-[8px]
     size-5;
 }
+
+.shortcutIndicator {
+  @apply invisible
+    absolute
+    right-2
+    ml-2
+    font-open-sans
```

Comment:
The `kbd`element overwrites the font style given here. Also, it may be better to use `font-ibm-plex-mono` instead of `font-open-sans`